media/static-bundles.json

```diff
@@ -1422,6 +1422,12 @@
       ],
       "name": "tos_modal_link"
     },
+    {
+      "files": [
+        "js/privacy/redirects.es6.js"
```

Comment:
client side because we are only redirecting urls with specific anchors and don't seem to have access to that for typical django redirects